parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -87,15 +91,33 @@ pub trait ConvertMessage {
 
 pub type CallIndex = [u8; 2];
 
-impl<CreateAssetCall> ConvertMessage for MessageToXcm<CreateAssetCall>
+pub trait XcmFeeGetter {
+	// Measures the maximum amount of gas a command will require
+	fn fee(command: &Command) -> u128;
+}
+
+/// A meter that assigns a constant amount of gas for the execution of a command
+pub struct ConstantFeeForInboundMessage;
+
+impl XcmFeeGetter for ConstantFeeForInboundMessage {
+	fn fee(command: &Command) -> u128 {
+		match command {
+			Command::RegisterToken { .. } => 2_000_000_000,
+			Command::SendToken { .. } => 1_000_000_000,
+		}
```

Comment:
We did lose a little bit of coverage with these new lines though I am not sure if we need tests for such a simple implementation.

Reply:
https://github.com/Snowfork/snowbridge/pull/970/commits/f8fe653d239e445e3ab44da4a590e71ace49bde7